list/list.json

```diff
@@ -532,12 +532,12 @@
     },
     {
       "name": "AnkeStrengt",
-      "description": "der Quahlnal"
+      "description": "der Quahlkanal"
```

Comment:
Wird das nicht ohne h geschrieben? 

@mrStronge war `Qualnal` eigentlich beabsichtigt? Es würde mich zumindest nicht wundern 😅

Reply:
Sicher war das Absicht. Ändere das bitte wieder zurück 😏

Reply:
Ah, Ich sehs gerade. Ja ohne „h“. Da kam wieder der legastheniker in mir durch 😏. Der Rest bleibt bitte

Reply:
```suggestion
      "description": "der Qualnal"
```